tests/phpunit/tests/media.php

```diff
@@ -4069,6 +4233,345 @@ public function data_special_contexts_for_the_content() {
 		);
 	}
 
+	/**
+	 * Tests that wp_get_loading_attr_default() returns the expected loading attribute value.
+	 *
+	 * @ticket 53675
+	 * @ticket 56930
+	 * @ticket 58235
+	 *
+	 * @covers ::wp_get_loading_optimization_attributes
+	 *
+	 * @dataProvider data_wp_get_loading_attr_default
+	 *
+	 * @param string $context
+	 */
+	public function test_wp_get_loading_optimization_attributes( $context ) {
+		$attr = array(
+			'width'  => 300,
+			'height' => 200,
+		);
```

Comment:
Please clarify why these specific width and height were chosen.
```suggestion
		// The product of width * height must be >50000 to qualify for high priority image.
		$attr = array(
			'width'  => 300,
			'height' => 200,
		);
```

Reply:
@kt-12 I have introduced two new utility methods in the test class so that we don't always have to manually declare the sizes to use: https://github.com/WordPress/wordpress-develop/pull/4495/commits/d27afa6dc698d3d975780dba16a171d66b4a04f3

WIth those functions it's a lot easier to follow and we don't have to always "reinvent" the correct sizes to use.